crates/runner-host/crates/web-client/README.md

```diff
@@ -0,0 +1,24 @@
+# Wasefire Web Client
+
+This is a fairly minimal web app for interacting with the runner that's built with Yew and [Trunk].
+
+### Running
+
+```bash
+trunk serve
```

Comment:
Yeah I believe something like that should be possible. Once truck build has been run it can be served just like a static page. I'll look into that for when we I add a flag to the runner